tests/test_maintel_lasertracker_align.py

```diff
@@ -0,0 +1,137 @@
+            }
+        )
+
+        self.script.laser_tracker.rem.lasertracker_1 = unittest.mock.AsyncMock()
+        self.script.laser_tracker.rem.lasertracker_1.configure_mock(
+            **{
+                "cmd_align.set_start": self.mock_laser_align,
+                "evt_offsetsPublish.next.side_effect": self.get_offsets,
+                "evt_laserStatus.aget": self.get_laser_status,
+                "evt_summaryState.aget": self.get_summary_state,
+                "evt_summaryState.next": self.get_summary_state,
+            }
+        )
+
+        self.state_0 = [1, 1, 1, 1, 1, 1]
+        self.laser_status = types.SimpleNamespace(status=LaserStatus.ON)
+        return (self.script,)
+
+    async def get_summary_state(self, *args, **kwargs):
+        return types.SimpleNamespace(summaryState=State.ENABLED)
+
+    async def mock_laser_align(self, *args, **kwargs):
+        await asyncio.sleep(0.5)
+
+    async def get_laser_status(self, *args, **kwags):
+        await asyncio.sleep(0.5)
+        return self.laser_status
+
+    async def apply_offsets(self, corrections, *args, **kwags):
+        await asyncio.sleep(0.5)
+        self.state_0 += corrections
+
+    async def get_offsets(self, *args, **kwags):
+        # return corrections to be non zero the first time this is called
+        await asyncio.sleep(0.5)
+        if any(self.state_0):
+            offsets = types.SimpleNamespace(
+                dX=0.5e-6,
+                dY=0.5e-6,
+                dZ=0.5e-6,
+                dRX=0.5,
+                dRY=0.5,
+            )
+        else:
+            offsets = types.SimpleNamespace(
+                dX=0.0,
+                dY=0.0,
+                dZ=0.0,
+                dRX=0.0,
+                dRY=0.0,
+            )
+
+        return offsets
+
+    async def test_configure(self):
+        # Try configure with minimum set of parameters declared
+        async with self.make_script():
+            max_iter = 10
+            target = "M2"
+            tolerance_linear = 1.0e-7
+            tolerance_angular = 5.0 / 3600.0
+
+            await self.configure_script(
+                max_iter=max_iter,
+                target=target,
+                tolerance_linear=tolerance_linear,
+                tolerance_angular=tolerance_angular,
+            )
+
+            assert self.script.max_iter == max_iter
+            assert self.script.target == getattr(AlignComponent, target)
+            assert self.script.tolerance_linear == tolerance_linear
+            assert self.script.tolerance_angular == tolerance_angular
+
+    async def test_run(self):
+        # Start the test itself
+        async with self.make_script():
+            await self.configure_script(
+                max_iter=10,
+                target="M2",
+                tolerance_linear=1.0e-7,
+                tolerance_angular=5.0 / 3600.0,
+            )
+
+            # Run the script
+            await self.run_script()
+
```

Comment:
You need some assertions here to check the script actually did what you expected it to do.